.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+## Proposed changes
+
+Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
+
+## Types of changes
```

Comment:
not really sure if section is needed since we have labels on prs

Reply:
you right, I'll update that